includes/REST_API/Font_Controller.php

```diff
@@ -250,11 +250,11 @@ public function get_items( $request ) {
 				array_push( $fonts, ...$this->get_builtin_fonts() );
 
 				// For custom fonts the searching will be done in WP_Query already.
-				if ( isset( $registered['search'], $request['search'] ) && ! empty( $request['search'] ) ) {
+				if ( isset( $registered['search'], $request['search'] ) && ! empty( $request['search'] ) && \is_string( $request['search'] ) ) {
 					$fonts = array_values(
 						array_filter(
 							$fonts,
-							static function( $font ) use ( $request ) {
+							static function( $font ) use ( $request ) { // phpcs:ignore SlevomatCodingStandard.Functions.RequireArrowFunction.RequiredArrowFunction
```

Comment:
Becuase of the comment used for phpstan typing below. Without this, it will convert to a arrow function and remove this comment.

Reply:
And why can't we use an arrow function here?

Reply:
A linter just warns and doesn't remove things.

Not getting any PHPCS & PHPStan warnings with this:

```php
array_filter(
	$fonts,
	/**
	 * Font data.
	 *
	 * @param array{family: string} $font
	 */
	static fn( array $font ) => (
		/**
		 * Request data.
		 *
		 * @var array{search: string} $request
		 */
		false !== stripos( $font['family'], $request['search'] )
	)
)
```

Reply:
Ditto for the other instances below

Reply:
When I ran npm run lint:php:fix, it did remove it.

Reply:
I pushed the changes in https://github.com/GoogleForCreators/web-stories-wp/pull/12672/commits/2cc18510754318d3bdd5972333ac5be9bfecf145. I am getting phpstan errors still. Hard to workout becuase of upstream error - https://github.com/phpstan/phpstan/issues/6454

Reply:
The error 
```
Parameter #2 $callback of function array_filter expects callable(mixed): mixed, Closure(array): bool given.
```

Reply:
@spacedmonkey 38e8d98284841bb253edf833de81f424289f817b fixes this.

PHPStan rightfully warned about `$fonts` no longer be an array of just `Font` arrays, because we didn't properly specify the return type of `get_custom_fonts()`. Adding `@phpstan-return Font[]` there resolves that.

No more warnings 🚀

Reply:
Thank you!